packages/docs/src/components/Playground.astro

```diff
@@ -107,7 +115,7 @@ const { url }: Props = {
   document.querySelectorAll('[data-type="set"]').forEach((element) => {
     element.addEventListener('click', () => {
       const commandName = element.getAttribute('data-command');
-      const value = JSON.parse(element.getAttribute('data-value') || '');
```

Comment:
This was throwing error when the value of `element.getAttribute('data-value')` was a string.